### PR TITLE
Add assertion for Business name in the candidate details (B+E)

### PR DESCRIPTION
**Pull request by naqvis5**
## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: test/e2e/step-definitions/journal-steps.ts [hunk 1]
```diff
@@ -191,6 +191,7 @@ Then('I continue the write up for {string}', (candidateName) => {
 const viewCandidateDetails = (candidateName) => {
   const buttonElement = getElement(by.xpath(`//h3[text()[normalize-space(.) = "${candidateName}"]]`));
   return clickElement(buttonElement);
+
```
Comment (turnerc1) [inline test/e2e/step-definitions/journal-steps.ts]: Remove white space

Comment (naqvis5) [reply test/e2e/step-definitions/journal-steps.ts]: done

Comment (naqvis5) [reply test/e2e/step-definitions/journal-steps.ts]: Done

File: test/e2e/step-definitions/journal-steps.ts [hunk 2]
```diff
@@ -191,7 +191,6 @@ Then('I continue the write up for {string}', (candidateName) => {
 const viewCandidateDetails = (candidateName) => {
   const buttonElement = getElement(by.xpath(`//h3[text()[normalize-space(.) = "${candidateName}"]]`));
   return clickElement(buttonElement);
-
```
Comment (JCarterDVSA) [inline test/e2e/step-definitions/journal-steps.ts]: don't know why it's doing this even after our call..

**Review by JCarterDVSA — APPROVED** [review]

**Review by TomBillingtonUK — APPROVED** [review]

**Pull request merged**